modules/host/contractmanager/persist.go

```diff
@@ -31,6 +32,34 @@ type (
 	}
 )
 
+func (sf *savedStorageFolder) Equals(sfb savedStorageFolder) bool {
+	if sf.Index != sfb.Index || sf.Path != sfb.Path || len(sf.Usage) != len(sfb.Usage) {
+		return false
+	}
+
+	for i := range sf.Usage {
+		if sf.Usage[i] != sfb.Usage[i] {
+			return false
+		}
+	}
+
+	return true
+}
+
+func (s *savedSettings) Equals(sb savedSettings) bool {
```

Comment:
no need to export this. Also, convention (in the `siad` repo, at least) is to add docstrings to all methods.